src/collections/_documentation/workflow/search.md

```diff
@@ -218,12 +217,22 @@ Below is a list of Event-level tokens reserved and known to Sentry:
 
 : Restrict results to events with a matching stack property.
 
-&nbsp;
-### Custom Tags
+### Custom Tags {#custom-tags}
```

Comment:
Let's leave out these guys: {#custom-tags}

They're automatic and only need to be included if you want an id that is different from the title, which we should avoid. The existing ones in the codebase were just there to prevent broken links from the old docs, but given that page scroll location isn't *that* critical, it might be worth just removing them universally.